package.json

```diff
@@ -3,11 +3,14 @@
   "version": "1.0.0",
   "description": "Electron app build using ReactJS and Parcel web application bundler",
   "main": "src/electron.js",
+  "targets": {
+    "main": false
+  },
   "scripts": {
-    "react-start": "parcel -p 3000 index.html --out-dir build",
-    "react-build": "parcel build index.html --out-dir build --public-url ./",
+    "react-start": "parcel -p 3000 index.html --dist-dir build",
+    "react-build": "parcel build index.html --dist-dir build --public-url ./",
     "electron-build": "electron-builder -mwl",
-    "clean-build": "rm -rf build/ .cache dist/",
+    "clean-build": "rm -rf build/ .cache .parcel-cache dist/",
     "build": "yarn clean-build && yarn react-build && yarn electron-build",
```

Comment:
I deleted `package-lock.json` and generated `yarn.lock` since these scripts only work with Yarn anyway.